frontend/src/scenes/hog-functions/configuration/HogFunctionTest.tsx

```diff
@@ -463,25 +456,11 @@ export function HogFunctionTest({ configurable }: { configurable: boolean }): JS
                                     pagination={{ pageSize: 200, hideOnSinglePage: true }}
                                 />
                             </div>
-                        ) : configurable ? (
+                        ) : (
                             <div className="deprecated-space-y-2">
                                 <LemonField name="globals">
                                     {({ value, onChange }) => (
                                         <>
-                                            {value.includes('"this_is_an_example_event": true') ? (
```

Comment:
Wasn't a fan of this banner as its not exactly accurate and misleading. Instead I moved the button to load a new event out of the dropdown